src/chrome/content/rules/du00.cc.xml

```diff
@@ -0,0 +1,21 @@
+<ruleset name="du00.cc">
+
+	<target host="m.du00.cc" />
+
+	<target host="www.du00.cc" />
+
+	<!--	MCB:The text could not show.	-->
+
+		<exclusion pattern="^http://www\.du00\.cc/read/\d+/\d+/\d+\.html" />
+  		<test url="http://www.du00.cc/read/39/39898/9502924.html" />
+  		<test url="http://www.du00.cc/read/47/47751/11503878.html" />
+  		<test url="http://www.du00.cc/read/67/67925/16855700.html" />
+  		<test url="http://www.du00.cc/read/70/70200/17351417.html" />
+
+		<exclusion pattern="^http://www\.du00\.cc/t/t\.php\?id=" />
```

Comment:
m. is currently down for me, but I thought that I also saw mixed active content there.